<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_wizard_answer">
    <sys_wizard_answer action="INSERT_OR_UPDATE">
        <active>true</active>
        <answer>Server Patching Exception</answer>
        <button_label/>
        <dependent_value/>
        <element/>
        <name>Server Patching Exception</name>
        <next_question/>
        <order>100</order>
        <payload_name/>
        <question display_value="Create a new task" name="Create a new task">48ecf8bf0a0110e9006e17d69550477e</question>
        <roles/>
        <script/>
        <sys_class_name>sys_wizard_answer</sys_class_name>
        <sys_created_by>XuHx</sys_created_by>
        <sys_created_on>2018-10-23 20:55:57</sys_created_on>
        <sys_customer_update>false</sys_customer_update>
        <sys_id>e0b5f70e131d2300339d30128144b086</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>Server Patching Exception</sys_name>
        <sys_package display_value="Global" source="global">global</sys_package>
        <sys_policy/>
        <sys_replace_on_upgrade>false</sys_replace_on_upgrade>
        <sys_scope display_value="Security Policy Exception Manager">399a87f013e82b4032eb7e276144b0d1</sys_scope>
        <sys_update_name>sys_wizard_answer_e0b5f70e131d2300339d30128144b086</sys_update_name>
        <sys_updated_by>XuHx</sys_updated_by>
        <sys_updated_on>2018-10-23 20:55:57</sys_updated_on>
        <table/>
        <target_url>x_inhe_spem_server_patching_exception.do</target_url>
        <type>1</type>
    </sys_wizard_answer>
</record_update>
